programs/perpetuals/src/state/multisig.rs

```diff
@@ -34,6 +34,7 @@ pub enum AdminInstruction {
     WithdrawFees,
     WithdrawSolFees,
     SetCustomOraclePrice,
+    SetPermissionlessOraclePubkey,
```

Comment:
it doesn't need to be here, this enum only for admin instructions

Reply:
ah my bad, remnant from the earlier standalone set-key instr!  removed